hubble/hubble.go

```diff
@@ -156,6 +159,18 @@ func NewK8sHubble(client k8sHubbleImplementation, p Parameters) *K8sHubble {
 }
 
 func (k *K8sHubble) Log(format string, a ...interface{}) {
+	if k.params.RedactHelmCertKeys {
+		formattedString := fmt.Sprintf(format+"\n", a...)
+		for _, certKey := range []string{
+			base64.StdEncoding.EncodeToString(k.certManager.CAKeyBytes()),
```

Comment:
This assumes that the CA key bytes were encoded with `base64.StdEncoding` and will break if the encoding changes in the future. I suggest factoring out `base64.StdEncoding.EncodeToString` into a separate `encodeCAKeyBytes` function and calling this function both here and where this function is called (i.e. at lines 361 and 362 of this file).